docs/src/BHM-example.md

```diff
@@ -130,29 +129,50 @@ println(t_strat)
 Finally, we can start the main FCIQMC loop:
 
 ```@example BHM-example
-r_strat = ReportDFAndInfo(k = k, projector = UniformProjector(), writeinfo = false) # hide
 df = lomc!(Ĥ,svec;
```

Comment:
This will make `df` a tuple, which is confusing. I suggest to change this to
```suggestion
_, df = lomc!(Ĥ,svec;
```

Reply:
I am putting in `df, state = lomc!(Ĥ,svec;` follwing the docstring